### DISCUSSION TITLE
ML kit landmark detection

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5556] [Feature request]
As I have seen, the landmark recognition for the ML Kit is not yet implemented. That being said I meant to ask, if there is some sort of ETA for this feature, or if it may take a significant amount of time, until it is being released.

**Comment by kroikie** [comment]
@Navil

The issue at https://github.com/flutter/flutter/issues/31350 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by emeraldsanto** [comment]
Just wondering what the status is on this? I've been meaning to build an app focused around landmark detection but it seems I'll have to use React Native as their wrapper already supports this.

**Comment by gregoriopalama** [comment]
Since I also need this feature, I'm actually working on it. I've got this working on iOS. I'll test it on Android too and add a PR. Hope I can get this done by the end of the next week.

**Comment by yasharma2301** [comment]
@gregoriopalama I saw your pull request. Any news from the team?

**Comment by gregoriopalama** [comment]
@yasharma2301 unfortunately, there's no update on the PR. Maybe @kroikie could help us: do you know if the issue is going to be checked at? Is there any plan to include the feature in the near future?
